Gulpfile.js

```diff
@@ -103,14 +92,18 @@ gulp.task("test", done =>
 {
     const lPathArgs = getArgs()["path"];
     const lFileArgs = getArgs()["file"];
+    process.env.ENV__LOGGING_LEVEL = "OFF";
+    let lArgs = [];
 
     if (lPathArgs !== undefined)
```

Comment:
We would ideally support both `--path` and `--file` being used together instead of `path` taking precedence. I.e. default to global path but if we restrict path then we will run `file` only within that path

Reply:
I think that's a good idea, but also not in the scope of this PR. I noticed this issue, but figured it existed before me, so it was probably intentional?

Reply:
I think it was just lazy originally, aqueduct supports files over path and this is path over files. Agreed, lets table this. Feel free to resolve this thread